pgmpy/models/BayesianModel.py

```diff
@@ -410,9 +412,9 @@ def active_trail_nodes(self, start, observed=None):
         >>> student.add_nodes_from(['diff', 'intel', 'grades'])
         >>> student.add_edges_from([('diff', 'grades'), ('intel', 'grades')])
         >>> student.active_trail_nodes('diff')
```

Comment:
@khalibartan @ankurankan Bayesian Model specifies nodes should be strings so I was using strings for single variable.
But there is another problem that add_edge method specifies its argument to be any hashable python objects. This is in contradiction with docstring of Bayesian Model.